kong/conf_loader/init.lua

```diff
@@ -581,7 +581,7 @@ local CONF_PARSERS = {
   max_queued_batches = { typ = "number" },
 
   wasm = { typ = "boolean" },
-  wasm_modules = { typ = "array" },
+  wasm_filters_path = { typ = "string" },
```

Comment:
Since we're leaning towards using the boolean `wasm = on|off` setting to toggle wasm, would it make sense to remove `wasm_filters_path` as a setting and use a standardized location (i.e. `{{KONG_PREFIX}}/wasm_filters`) instead?

We can always add `wasm_filters_path` later on if there is a compelling case to use a non-default location.

@locao what do you think?

Reply:
I don't have a strong opinion on this. I like a lot of knobs to adjust even if they were never touched, but that's not how a good product is built 😅 

For the sake of keeping things simpler, I'd say let's try without `wasm_filters_path` and see if we miss using it during development phase.

Reply:
I can imagine it being kind of annoying for our integration tests if the path is not configurable, _but_ in my opinion this gives us the incentive to make sure symlinks are supported. Ideally something like this should be supported out-of-the box:

```sh
ln -s \
  /path/to/git/kong/spec/fixtures/wasm/a/b/c/d/my_filter.wasm \
  /path/to/my_kong_prefix/wasm_filters/my_filter.wasm
```